src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/Collection.orm.xml

```diff
@@ -5,6 +5,28 @@
                   xmlns:gedmo="http://gediminasm.org/schemas/orm/doctrine-extensions-mapping">
 
     <entity name="Sulu\Bundle\MediaBundle\Entity\Collection" table="me_collections" repository-class="Sulu\Bundle\MediaBundle\Entity\CollectionRepository">
```

Comment:
this is not configured as a `mapped-superclass` right now. do we need to change this?